kong/clustering/rpc/manager.lua

```diff
@@ -188,10 +189,26 @@ function _M:_handle_meta_call(c, cert)
   assert(type(info.kong_hostname) == "string")
   assert(type(info.kong_conf) == "table")
 
+  local version = info.kong_version
```

Comment:
Could we add some comments to explain this (for other reviewers)?

Reply:
A comment was added.